tests/countries/test_trinidad_and_tobago.py

```diff
@@ -0,0 +1,270 @@
+            "2020-11-14",
+            "2021-11-04",
+            "2022-10-24",
+            "2023-11-12",
+            "2024-10-31",
+            "2025-10-20",
+        )
+        self.assertHolidayName(name, range(1963, 2050))
+
+    def test_eid_al_fitr(self):
+        name = "Eid-Ul-Fitr"
+        self.assertHolidayName(
+            name,
+            "2012-08-19",
+            "2013-08-08",
+            "2014-07-29",
+            "2015-07-18",
+            "2016-07-06",
+            "2017-06-26",
+            "2018-06-15",
+            "2019-06-05",
+            "2020-05-24",
+            "2021-05-13",
+            "2022-05-02",
+            "2023-04-22",
+            "2024-04-10",
+            "2025-03-31",
+        )
+        self.assertHolidayName(name, range(2012, 2025))
+
+    def test_2024(self):
+        self.assertHolidays(
+            TrinidadAndTobago(years=2024),
+            ("2024-01-01", "New Year's Day"),
+            ("2024-02-12", "Carnival Monday"),
+            ("2024-02-13", "Carnival Tuesday"),
+            ("2024-03-29", "Good Friday"),
+            ("2024-03-30", "Spiritual Baptist Shouter Liberation Day"),
+            ("2024-04-01", "Easter Monday"),
+            ("2024-04-10", "Eid-Ul-Fitr"),
+            ("2024-05-30", "Corpus Christi; Indian Arrival Day"),
+            ("2024-06-19", "Labour Day"),
+            ("2024-08-01", "African Emancipation Day"),
+            ("2024-08-31", "Independence Day"),
+            ("2024-09-24", "Republic Day"),
+            ("2024-10-31", "Diwali"),
+            ("2024-12-25", "Christmas Day"),
+            ("2024-12-26", "Boxing Day"),
+        )
+
+    def test_2025(self):
+        self.assertHolidays(
+            TrinidadAndTobago(years=2025),
+            ("2025-01-01", "New Year's Day"),
+            ("2025-03-03", "Carnival Monday"),
+            ("2025-03-04", "Carnival Tuesday"),
+            ("2025-03-30", "Spiritual Baptist Shouter Liberation Day"),
+            ("2025-03-31", "Eid-Ul-Fitr"),
+            ("2025-04-18", "Good Friday"),
+            ("2025-04-21", "Easter Monday"),
+            ("2025-05-30", "Indian Arrival Day"),
+            ("2025-06-19", "Corpus Christi; Labour Day"),
+            ("2025-08-01", "African Emancipation Day"),
+            ("2025-08-31", "Independence Day"),
+            ("2025-09-24", "Republic Day"),
+            ("2025-10-20", "Diwali"),
+            ("2025-12-25", "Christmas Day"),
+            ("2025-12-26", "Boxing Day"),
+        )
+
+    def test_l10n_default(self):
+        self.assertLocalizedHolidays(
+            ("2025-01-01", "New Year's Day"),
+            ("2025-03-03", "Carnival Monday"),
+            ("2025-03-04", "Carnival Tuesday"),
+            ("2025-03-30", "Spiritual Baptist Shouter Liberation Day"),
+            ("2025-03-31", "Eid-Ul-Fitr"),
+            ("2025-04-18", "Good Friday"),
+            ("2025-04-21", "Easter Monday"),
+            ("2025-05-30", "Indian Arrival Day"),
```

Comment:
Looks like Labour Day got forgotten here:
```suggestion
            ("2025-04-21", "Easter Monday"),
            ("2025-05-01", "Labour Day"),
            ("2025-05-30", "Indian Arrival Day"),
```

Reply:
According to [Wiki](https://en.wikipedia.org/wiki/Labour_Day#:~:text=In%20Trinidad%20and%20Tobago%2C%20Labour%20Day%20is%20celebrated%20every%2019%20June.) Labour Day falls on "2025-06-19" for TT.

Reply:
Good catch